inc/thorvg.h

```diff
@@ -1039,10 +1047,10 @@ class TVG_EXPORT Picture final : public Paint
  *
  * @brief A class to composite children paints.
  *
- * As the traditional graphics rendering method, TVG also enables scene-graph mechanism,
+ * As the traditional graphics rendering method, TVG also enables scene-graph mechanism.
  * This feature supports an array function for managing the multiple paints as one group paint.
  *
- * As a group, scene can be transformed, translucent, composited with other target paints,
+ * As a group, the scene can be transformed, made opaque and composited with other target paints,
```

Comment:
We intent to say "translucent." not opaque

Reply:
done